chapters/api_concepts.txt

```diff
@@ -242,29 +242,36 @@ brevity:
 [[api_concepts_introspection]]
 === Object Introspection
 
-ANARI supports introspection of objects, i.e., querying supported object
+ANARI supports introspection of objects, i.e., querying supported features, object
 subtypes and information about their parameters, which is particularly useful
 to enumerate and inspect implementation-specific object features. The intended
 use is to allow for building a graphical user interface for, e.g., specific
 renderers or materials, and to provide hints for the debug layer to aid in
-validating features.  Because the meta-information of objects and parameters
-supported by devices is static, it can be queried without creating a device
-first by passing the device name.
+validating features.
+
+Device subtypes and their implemented features can be queried
+from a library before instantiating a device. Object and parameter specific
+information can be queried from an instantiated device object.
 
 [source,cpp]
 ----
 const char **anariGetDeviceSubtypes(ANARILibrary);
+const char **anariGetDeviceFeatures(ANARILibrary, const char *deviceSubtype);
```

Comment:
let's just move after the paragraph (directly before it's description)

Reply:
Moved in eaac0f744a138d5635c3636e59e8d1c2cb4e12a0